org.thingpedia.iot.speaker/manifest.tt

```diff
@@ -0,0 +1,61 @@
+abstract class @org.thingpedia.iot.speaker
+#_[thingpedia_name="Speaker"]
+#_[thingpedia_description="Interface for Speaker."]
+#_[description="Interface for Speaker."]
+#[license="CC-0"]
+#[license_gplcompatible=true]
+#[subcategory="home"]
+{
+  monitorable query state(out state : Enum(on,off))
+  #_[canonical="switch state"]
+  #_[confirmation="the state of $__device"]
+  #_[formatted=["It is ${state}"]];
+
+  action set_power(in req power: Enum(on,off) #_[prompt="Do you want to turn it on or off?"] #_[canonical="power"])
+  #_[canonical="turn your speaker on or off"]
+  #_[confirmation="turn ${power} $__device"]
+  #[confirm=true];
+  #[doc="turn on or off the TV"];
+
+  action raise_volume()
+  #_[canonical="increase speaker volume"]
+  #_[confirmation="increase $__device's volume"]
+  #[confirm=true];
+  #[doc="increase the volume of the tv"];
+
+  action lower_volume()
+  #_[canonical="decrease speaker volume"]
+  #_[confirmation="decrease $__device's volume"]
+  #[confirm=true];
+  #[doc="decrease the volume of the tv"];
+
+  action mute()
+  #_[canonical="mute speaker"]
+  #_[confirmation="mute $__device"]
+  #[confirm=true];
+  #[doc="mute the tv"];
+
+  action unmute()
+  #_[canonical="unmute speaker"]
+  #_[confirmation="unmute $__device"]
+  #[confirm=true];
+  #[doc="unmute the tv"];
+
+  action set_volume(in req volume: Number #_[prompt="From 0 to 100, what volume do you want to set?"] #_[canonical="volume"])
+  #_[canonical="set volume for speaker"]
+  #_[confirmation="set $__device's volume to $volume"]
+  #[confirm=true];
+  #[doc="set the tv volume"];
+
+  action set_sink()
+  #_[canonical="set audio sink on speaker"]
+  #_[confirmation="set $__device to be the default audio output"]
+  #_[confirmation_remote="set $__person's $__device to be their default audio output"]
+  #[doc="set this speaker as the default audio output device"];
+
+  action play_music()
+  #_[canonical="play music on bluetooth speaker"]
```

Comment:
No more "bluetooth" in the generic speaker interface

Reply:
Fixed in latest commit